index.html

```diff
@@ -25,9 +25,26 @@
 
 <!--<div id="overlayGIF"></div> overlayGIF details in main.css-->
     <div id="info-overlay">
+    
         <div id="text">
-            <B>THIRTY SECOND STORIES</B><BR><BR>
+            <div id="title">THIRTY SECOND STORIES</div><p>
```

Comment:
I added that (following Lexy's model, probably foolishly!) and it's just for the Title at the top of the info-overlay, which is bigger than the rest of the text.